compiler/luci/pass/src/ResolveCustomOpMaxPoolWithArgmaxPass.cpp

```diff
@@ -784,6 +823,8 @@ namespace luci
  *   |  [Mul 1/<window width>]
  *   |                \
  *   |              [Floor]
+ *   |                 |
+ *   |            [Requantize]
```

Comment:
Code seems to be `CircleDepthwiseConv2D` but `Requantize` here. Is  `Requantize` a meaning of a purpose?
May this?
```
[DepthwiseConv2D] for requantize
```

Reply:
Yes, this is DepthwiseConv2D, but it serves requantization (aka rescale) purpose.

I've added `[Requantize]` just for simplicity, because it is not really matter what nop operation to use (unless it is not wiped out by optimizations)

If you think this matters I propose to merge this change, plus https://github.com/Samsung/ONE/pull/7517/files#r691675340 in batch in new PR.
What do you think?

Reply:
(1) Using `Requantize` may make confusion understanding and also only `DepthwiseConv2D` may make hard to understand the purpose of inserted op. Like existing nodes, `[DepthwiseConv2D for requantize]` seems OK.

(2) Personnally I do not like to mix multiple contexts in one PR. It makes understanding difficult.